### PR TITLE
[SPCGear GK630K] Initial support

### PR DESCRIPTION
**Pull request by ppkt**
## Description

In this PR I'm adding initial support for [SPC Gear GK630K](https://spcgear.com/en/product/gk630k-tournament-kailh-brown-rgb/) keyboard. Basically, it's the same keyboard as Redragon K552 rev2 but I had to set a AHB prescaler value to 0x02 (instead of 0x00 in K552). With 0x00 QMK reported incorrectly keys pressed in top row - for example pressing [Esc] was reported as [Esc] + [F1], pressing [F6] as [F6] and [F7] - weird stuff.

RGB works very well, with K552 firmware I was also able to control LEDs from Open RGB but I think green and blue channels should be swapped - I have to investigate this in a free time.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* #127 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/spcgear/gk630k/config.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x3299
+#define PRODUCT_ID      0x4e5e
```
Comment (ppkt) [inline keyboards/spcgear/gk630k/config.h:23]: Vendor and product IDs are taken from original firmware, I noticed other SPC keyboards in this repo are using different values - 0C45:8508 - should I change this?

Comment (dexter93) [reply keyboards/spcgear/gk630k/config.h:23]: We usually define those as in the original firmware. If keyboard in stock state comes with those, leave them in. Unless there are driver conflicts

**Comment by ppkt** [comment]
I guess this PR can be closed in favor of #207

**Comment by davecarrijo** [comment]
Just to be sure, the Bluetooth as implemented in the late updates?

**Pull request closed without merging**